src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  movies: Movie[];
+  setMovies: (movie: Movie[]) => void;
+  query: string;
+  setQuery: (query: string) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({
+  movies,
+  setMovies,
+  query,
+  setQuery,
+}) => {
+  const [loadData, setLoadData] = useState(false);
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [loadedMovie, setLoadedMovie] = useState<Movie | null>();
+
+  const loadMovie = async () => {
+    setLoadData(true);
+
+    try {
+      const movieFromServer = await getMovie(query);
+
+      setLoadData(false);
+
+      if ('Error' in movieFromServer) {
+        throw new Error('Can&apos;t find movie such a title');
+      } else {
+        if (movieFromServer.Poster === 'N/A') {
+          movieFromServer.Poster
+            = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+        }
```

Comment:
This case is better to handle inside the `MovieCard` component, not mutate API response.